lib/features/thread/presentation/thread_controller.dart

```diff
@@ -666,7 +666,9 @@ class ThreadController extends BaseController with EmailActionController {
         _searchEmailFilter.sortOrderType.isScrollByPosition(),
         0,
       ),
-      beforeOption: const None(),
+      beforeOption: !_searchEmailFilter.sortOrderType.isScrollByPosition()
+          ? const None()
```

Comment:
not a good idea in case: 
- user search in a range of time, but when it refreshes, `before` is not applied

Reply:
@hoangdat  Please attach error video here.

Reply:
Can we verify `SearchEmailFilter` in some cases:
- load more: scroll by position and not by position (before filter?)
- refresh: scroll by position and not by position (before filter?)